doc/source/user_guide/hpc_slurm.rst

```diff
@@ -296,6 +300,12 @@ The expected output of the job should be:
     Number of CPUs: 10.0
 
 
+Examples
+========
+
+In :ref:`hpc_ml_ga_example` an example which uses machine learning genetic algorithm in
+an HPC system managed by SLURM scheduler can be found.
```

Comment:
```suggestion
For an example that uses a machine learning genetic algorithm in
an HPC system managed by SLURM scheduler, see :ref:`hpc_ml_ga_example`.
```
Perhaps you can use GA here for genetic algorithm. (I don't know if it's been defined before its use here. Once defined, you can use GA thereafter.